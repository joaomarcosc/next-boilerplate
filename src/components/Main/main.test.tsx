import { render, screen } from '@testing-library/react'
import { Main } from '.'

describe('<Main />', () => {
  it('Should be render heading', () => {
    const { container } = render(<Main />)

    const heading = screen.getByRole('heading', { name: /react avan√ßado/i })

    expect(heading).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})
