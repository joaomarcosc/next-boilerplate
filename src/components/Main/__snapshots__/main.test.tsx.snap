// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main /> Should be render heading 1`] = `
<h1>
  React Avan√ßado
</h1>
`;
